src/main/java/org/testng/internal/Parameters.java

```diff
@@ -102,7 +102,7 @@
   public static Object[] createConfigurationParameters(Method m,
       Map<String, String> params,
       Object[] parameterValues,
-      @Nullable ITestNGMethod currentTestMethod,
```

Comment:
:+1: Sure! I've just removed it because a custom `@Nullable` is not supported be IDE.
I will use `javax.annotation.Nullable` instead.